### ISSUE TITLE
[Feature Request] Compare against some faster Python web frameworks

### ISSUE THREAD

**Issue by jordangarside** [sparckles/Robyn#193]
Hi there 👋 ,

It’d be interesting to see [Vibora](https://github.com/vibora-io/vibora) and [Japronto](https://github.com/squeaky-pl/japronto) in your performance comparisons.

They’re not really maintained, but their performance is in a different class from things like uvicorn/django/flask.

**Comment by sansyrox** [comment]
Hi @jordangarside ,

I just saw these projects. I will definitely have a look the comparison. Thank you! :D

**Comment by sansyrox** [comment]
Hi @jordangarside ,

I tried installing these projects on my machine but I was unable to get them up and running. I remember trying with 3.10 and 3.9 for sure and maybe even 3.8 .

Which Python version do you use them wiith?

**Comment by jordangarside** [comment]
Yeah it looks like I also had issues with the pypi installs for both,

For Vibora [try this](https://github.com/vibora-io/vibora/issues/195#issuecomment-423196635):
```
pip uninstall vibora
pip install Cython
git clone https://github.com/vibora-io/vibora.git
cd vibora
python build.py
python setup.py install
```

For Japronto [try this](https://github.com/squeaky-pl/japronto/issues/104#issuecomment-431665470):
```
pip install https://github.com/squeaky-pl/japronto/archive/master.zip
```

**Comment by sansyrox** [comment]
Thanks @jordangarside . I will give it a go :D
